src/hooks/useSetActiveAddress.ts

```diff
@@ -1,8 +1,12 @@
-import { useEffect, useState } from 'react';
+import { useEffect, useRef, useState } from 'react';
 import { DefaultAccount, AccountValue } from 'src/types/defaultAccount';
 
-function useSetActiveAddress(defaultAccount: DefaultAccount) {
+function useSetActiveAddress(
+  defaultAccount: DefaultAccount,
+  noUpdate?: boolean
+) {
```

Comment:
don't understand why need this hook, if we have redux and selectors like this `selectCurrentAddress`?

Reply:
<img width="742" alt="image" src="https://github.com/cybercongress/cyb-ts/assets/18665326/af5f9d48-c533-4d5d-8b0f-591b911ca984">